protos/sources/vega/governance.proto

```diff
@@ -66,6 +101,12 @@ message NewMarketConfiguration {
   string quadratic_slippage_factor = 10;
 }
 
+// New spot market on Vega
+message NewSpotMarket {
+  // Configuration of the new spot market.
+  NewSpotMarketConfiguration changes = 1;
+}
+
 // New market on Vega
 message NewMarket {
   // Configuration of the new market.
```

Comment:
Now that there's spot, I wonder if this string should be more specific about what kind of market

Reply:
maybe, but still for a while at least it will remain that Market == FutureMarket and SpotMarket == SpotMarket